lib/fathead/mdn_css/output.txt

```diff
@@ -375,23 +543,52 @@
-active	R	:active										
+CSSRule	R	cssrulelist										
+CSSRule.selectorText	R	cssstylerule.selectortext										
+CSSRuleList	R	cssrulelist										
+CSSStyleSheet	R	cssrule.csstext										
+U+2026 Horizontal Ellipsis	R	text-overflow										
+[ <line-names>? <string> <track-size>? <line-names>? ]+ [ / <explicit-track-list> ]?	A										<section class="prog__container"><p>Sets grid-template-areas to the strings listed, grid-template-rows to the track sizes following each string (filling in auto for any missing sizes), and splicing in the named lines defined before/after each size, and grid-template-columns to the track listing specified after the slash (or none, if not specified).\n  \n Note: The repeat() function isn’t allowed in these track listings, as the tracks are intended to visually line up one-to-one with the rows/columns in the “ASCII art”.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template#Values
+[ [ | infinite ]{2} ]#	A										<section class="prog__container"><p>Defines a comma-separated list of ranges. For each individual range, the first value is the lower bound and the second value is the upper bound. A range is inclusive, that means it always contains both, the lower and upper bound numbers. If infinite is used as the first value in a range, it represents negative infinity; if it is used as the second value, it represents positive infinity. The range of the counter style is the union of all the ranges defined in the list.\n If the lower bound of any range is higher than the upper bound, the entire descriptor is invalid and will be ignored.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style/range#Values
+[ auto-flow && dense? ] <grid-auto-rows>? / <grid-template-columns>	A										<section class="prog__container"><p>Sets up an auto-flow by setting the column tracks explicitly via the grid-template-columns property (and the grid-template-rows property to none) and specifying how to auto-repeat the row tracks via grid-auto-rows (and setting grid-auto-columns to auto). grid-auto-flow is also set to row accordingly, with dense if it’s specified.\n All other grid sub-properties are reset to their initial values.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/grid#Values
+[<string> <string>]+	A										<section class="prog__container"><p>One or more pairs of <string> values for open-quote and close-quote. The first pair represents the outer level of quotation, the second pair is for the first nested level, next pair for third level and so on.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/quotes#Values
+a	A										<section class="prog__container"><p>Is an <angle> representing the angle of the rotation. A positive angle denotes a clockwise rotation, a negative angle a counter-clockwise one.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate3d#Values
```

Comment:
Again, I think we probably shouldn't capture this information, I'm not sure it's going to trigger, or that if it does it will be relevant.